packages/android-client/.release-it.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
do we still use release-it for android releases?

Reply:
another thing to discuss